server/README.md

```diff
@@ -149,19 +153,24 @@ Then run the following commands to install Hatohol:
 
 ## How to start
 (0.1) Run MySQL server for storing configuration.
+Type the following comannds to run MySQL server.
+
+    # system mysql start
 
 (0.2) Set QPid's setting
 (0.2.a) Disable authentification
 Add the following line in /etc/qpid/qpidd.conf
+In line 33, change #auto=yes to
 
     auth=no
 
 (0.2.b) Allow all yours (Setting of ACL)
 Example of /etc/qpid/qpidd.acl
+Change line 33.
```

Comment:
Now Hatohol discards Qpid support.
So we should remove the descriptions about it.